documentation/library_guide/overview.rst

```diff
@@ -46,7 +46,12 @@ Some APIs of the library may require a higher version of C++.
 To call Parallel API with the C++ standard policies, you need to install the following software:
 
 * A C++ compiler with support for OpenMP* 4.0 (or higher) SIMD constructs
-* |onetbb_long| or |tbb_long| 2019 and later
+* Depending on what parallel backend you want to use install either:
+
+  * |onetbb_long| or |tbb_long| 2019 and later
+  * A C++ compiler with support for OpenMP 4.5 (or higher)
+
+For more information about parallel backends please see :doc:`Execution Policies <parallel_api/execution_policies>`
```

Comment:
Please change to:
For more information about parallel backends, see :doc:`Execution Policies <parallel_api/execution_policies>`

We don't use "please" in customer facing documentation.

Reply:
done